src/xdocs/config_imports.xml

```diff
@@ -1726,21 +1800,52 @@ import java.util.stream.IntStream;
   &lt;property name="message" value="^'java\..*'.*"/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+import static javax.swing.WindowConstants.*; // Violation; all static imports comes at bottom
 
-        <p>To configure the check so that it matches default NetBeans formatter configuration
-           (tested on v8):</p>
+import java.net.URL; // Violation; Wrong order for 'java.net.URL'
+import java.security.KeyManagementException;
+import javax.net.ssl.TrustManager; // Ok; no blank line between "javax" and "java".
+
+import static java.awt.Button.A; // Violation;  Extra separation in import group
+import static java.lang.Math.PI; // Ok
+import static java.lang.Math.abs; // Ok, alphabetical case sensitive ASCII order, 'P' &lt; 'a'
```

Comment:
Why are some `// Ok;` and some `// Ok,`?
Same for Violation (plus `:`).  I would stay with `// Ok,` and `// Violation,`.

Reply:
Done

Reply:
Resolved.